examples/hello-world/hello-fedavg-numpy/requirements.txt

```diff
@@ -0,0 +1 @@
+nvflare~=2.4.0rc
```

Comment:
Should be 2.5.0rc

Reply:
This needs to be updated everywhere so maybe we do it in a PR just for updating the version of this?